app/mlr/src/mlr_engine.cpp

```diff
@@ -382,8 +389,11 @@ std::string MLREngine::PrintOneEval(int32_t ith_eval) {
 std::string MLREngine::PrintAllEval(int32_t up_to_ith_eval) {
   std::stringstream output;
   if (perform_test_) {
-    output << "Epoch Batch Train-0-1 Train-Entropy Num-Train-Used Test-0-1 "
-      << "Num-Test-Used Time" << std::endl;
+    output << "Epoch Batch Train-0-1 Train-Entropy Num-Train-Used Test-0-1 ";
+    if (perform_test_acc_) {
+      output << "Test-0-1-Acc";
```

Comment:
I think for user's convenience, we should put accuracy there, rather than 0-1-acc.